uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestRightsTest.php

```diff
@@ -115,7 +115,7 @@ public function testCanChangeDatesForReturnsFalseForTOILAndLeaveRequestTypesWhen
       $managerRightsService->canChangeDatesFor($contactID, $status, LeaveRequest::REQUEST_TYPE_LEAVE)
     );
 
-    $this->assertFalse(
+    $this->assertTrue(
```

Comment:
Since an admin can now change the dates of a TOIL/Leave request. 
We need to break down this test because the test is actually asserting False and also True:
- Test that a manager cannot change dates for Leave/TOIL request types.
- Test that an admin can change the dates for Leave/TOIL request types.
You can rename the current test to maybe `testCanChangeDatesForReturnsFalseForTOILAndLeaveRequestTypesWhenCurrentUserIsLeaveManagerIrrespectiveOfStatusPassed`
Add another test for `testCanChangeDatesForReturnsTrueForTOILAndLeaveRequestTypesWhenCurrentUserIsAdminIrrespectiveOfStatusPassed`. These are just suggestions for the names.